capstone/src/main/java/com/step902020/capstone/EventController.java

```diff
@@ -74,12 +73,16 @@
         noPastEvents.add(e);
       }
     }
+
+    // order by rank
+    Collections.sort(noPastEvents, (a, b) -> Integer.compare(b.getRank(), a.getRank()));
     return noPastEvents;
```

Comment:
It seems like the events don't print in rank order on the index page. I tried refreshing and the events would display in different orders each time and close, but not exact rank order

Reply:
Yeah I tested this feature, the events all return in correct order on the backend. I think it has to do with the order that the divs are placed and the way they "float".